pom.xml

```diff
@@ -1545,6 +1557,7 @@
 		<module>hibernate-dialect</module>
 		<module>eclipselink-platform</module>
 		<module>jboss-integration</module>
+		<module>swagger-integration</module>
```

Comment:
How is swagger-integration used? If it is for testing this needs to be in "test-integration-common", you need the bootstrap servlet then this needs to part of the byte code generation code.